packages/url_launcher/url_launcher_android/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## NEXT
+## 6.3.0
 
+* Implement taking in `BrowserConfiguration` parameter
```

Comment:
These need to follow the CHANGELOG style guide linked from the PR checklist (verb form, punctuation).